### ISSUE TITLE
RGB image for visualisation

### ISSUE THREAD

**Issue by tischi** [embl-cba/fiji-plugin-cats#9]
@davodogster
> Another suggestion which would be extremely helpful - allow the user to have an RGB image as the first layer in the multi layer stack so that they can annotate it with confidence ( annotating an HSB/LAB stack etc can be difficult without colour )
But when computing the features, obviously don’t compute the features for the first layer because it’s not in grayscale. I hope you understand what I mean here as I think something like this will be really helpful.

**Comment by tischi** [comment]
I think this is not possible, because ImageJ does not allow an ImageStack that is a mixture of RGB and other pixel types. But you can change the viewing mode of your multi-channel stack to "Composite". Then it will look like an color image (you can even adjust the LUT and B&C of each channel):

<img width="554" alt="image" src="https://user-images.githubusercontent.com/2157566/57844739-f511d400-77d0-11e9-85d1-2657f7806383.png">

**Comment by davodogster** [comment]
I have an idea which you can hopefully implement
How about one annotates the RGB/False colour image with the ROIS and adds all of them to the ROI manager.

Then can you program CATS so that I can add all these ROIS (e.g all for class 1) to the CATS stack from the ROI manager (maybe all at once after the ROIS are combined?) and finally then press the shortcut keys in CATS to assign these ROIs to a class.

This would be an awesome feature. 
Sam

**Comment by tischi** [comment]
This should be possible, but could you first explain me why the composite image view is not sufficient? This would be much easier and is a core feature of ImageJ.

**Comment by davodogster** [comment]
Thanks for the tip - it helps a bit for visualizing, although it's a weird false colour image. So this doesn't alter any of the channel values, it only alters the way we visualize the channels?

**Comment by tischi** [comment]
> So this doesn't alter any of the channel values, it only alters the way we visualize the channels?

Yes. You should be able to configure it such that it looks exactly like the RBG image. It's a bit hard to explain without showing you in person...

**Comment by tischi** [comment]
I think with the script that I wrote for you this is solved, is it?

**Comment by davodogster** [comment]
Sure Is solved

Get Outlook for iOS<https://aka.ms/o0ukef>
________________________________
